protocols/network.md

```diff
@@ -74,35 +74,35 @@ specifies a *scheme* called `HTTP` which tells the client that it should try
 TCP port number 80 on the server by default. If the URL uses `HTTPS` instead,
 the default port number is 443.
 
-The URL can optionally provide another, custom, port number but if nothing
-special is specified, it will use the default port for the scheme used in the
+The URL can optionally provide another, custom port number, but if nothing
+else is specified it will use the default port for the scheme used in the
```

Comment:
This is better but I think we can improve it further. How about:

"The URL can provide a custom port number, but if not specified the client will instead use the default port for the scheme used in the"

Reply:
I'll incorporate your suggestion, along with one modification: 

"The URL can provide a custom port number, but if one isn't specified the client will use the default port for the scheme used in the..."

Reply:
rather, I changed it to:

"The URL can include a custom port number. If a port number isn't specified, the client will use the default port for the scheme used in the URL."

Reply:
Sure, sounds fine. But without the contraction please!

Reply:
done!